README.md

```diff
@@ -960,6 +960,8 @@ Displays an input field which can accept input masking, asynchronous validation,
 - `width`: *optional* string to specify the width of the field. Valid options are: "full", "wide", "half", and "narrow". Default is "full".
 - `spacing`: *optional* string to specify the vertical spacing beneath each input. Valid options are: "compact", "loose", and "tight". Default is "loose".
 - `value`: *optional* string for input's initial value. Default is "".
+- `onEnter`: *optional* callback function for onEnter event.
+- `animateLabel`: *optional* makes label a large hint text when there is no value on input is blured (default false)
```

Comment:
I'm assuming animateLabel is bool? Can you please add `boolean` after *optional*?